models/modelsmodule.h

```diff
@@ -27,6 +27,8 @@
 #include <string>
 
 // Includes from sli:
+#include "connector_model.h"
+#include "genericmodel.h"
```

Comment:
From the changes in this file, I don't see why you would need to add these two includes here. Remove if not needed, or move to `modelsmodule.cpp` if needed there. They are also misplaced here under heading `Includes from sli`.